client/blocks/inline-help/style.scss

```diff
@@ -68,19 +62,6 @@
 		background: var( --color-primary );
 		border-color: var( --color-primary-40 );
 	}
-
-	&.has-notification::before {
```

Comment:
Confirmed to be unused. Seems like we can also remove it from here:

https://github.com/Automattic/wp-calypso/blob/7fcaec0432d9d76c548f2fe998025d691a7f79ea/client/assets/stylesheets/directly.scss#L160

Reply:
That seems to be a very different place, inside the Directly widget's markup. There's a `.thread-view-title-container.has-notification` class, displaying notifications on a discussion thread?